networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/ActiveEth2Network.java

```diff
@@ -123,11 +132,21 @@ public BeaconChainMethods getBeaconChainMethods() {
 
   @Override
   public void subscribeToAttestationCommitteeTopic(final int committeeIndex) {
+    if (state.get() != State.RUNNING) {
+      LOG.warn(
+          "Attestation committee can not be subscribed due to Gossip Managers not being initialized");
+      return;
+    }
     attestationGossipManager.subscribeToCommitteeTopic(committeeIndex);
```

Comment:
I think these GossipManagers need to be `volatile`.  And we can just check if they're null.

Reply:
I've reverted to using an atomic boolean instead. I believe that also solves this issue.

Reply:
I think `attestationGossipManager` etc still need to be `volatile` ...

Reply:
Now that the PR https://github.com/PegaSysEng/teku/pull/1679 is merged in, and the JIRA issue [BC-394](https://pegasys1.atlassian.net/browse/BC-394 ) is dealt with, this shouldn't be a big issue anymore. I've reverted the changes I made in `ActiveEth2Network` around gossip managers waiting for the store to initialize.